style.css

```diff
@@ -0,0 +1,217 @@
+*{
+    margin: 0;
+    padding: 0;
+    background-color: #000;
+    color: #fff;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    font-family: 'Comforter', cursive;
+    margin-left: 8px;
+}
+
+.logo__lyrics {
+    font-size: 2.5em;
+    color: rgb(219, 32, 22);
+}
+
+.mainLogo{
+    width: 3.5em;
+    margin: 6px 1px
+}
+
+.search {
+    display: flex;
+}
+
+.searchbar {
+    width: 40em;
+    height: 2.5em;    
+}
+
+.buttons {
+    font-family: "Font Awesome 5 Free"; font-weight: 900;
+    height: 2.8em;
+    width: 3.5em;
+}
+
+.option {
+    font-size: 1.5em;
+    background-color: transparent;
+    border: none;
+}
+
+.account{
+    font-size: 1.8em;
+    color: rgb(219, 32, 22);
+}
+
+.page{
+    display: flex;
+    margin: 0em 8em;
+    justify-content: space-between;
+}
+
+article {
+    width: 75%;
+}
+
+.menu {
+    display: flex;
+    width: 20em;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+video {
+    width: 100%;
+}
+
+.title {
+    font-size: 25px;
+}
+
+.media_info {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+aside{
+    display: flex;
+    flex-direction: column;
+    width: 25%;
+    margin-left: 15px;
+}
+.links{
+    list-style: none;
+    display: flex;
+    overflow-x: auto;
+    height: 60px;
+    align-items: center;
+    margin-bottom: 15px;
+       
+}
```

Comment:
Inside this selector, you left an empty line. Please make sure to remove it. Once you set up Linters, you will be able to find and correct these situations.